stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -215,6 +214,8 @@ spec:
         - containerPort: {{ .Values.artifactory.internalPort }}
         volumeMounts:
       {{- if .Values.artifactory.userPluginSecrets }}
+        - name: tmp-plugins
```

Comment:
I think this mount is not needed as we have the `/tmp/plugins` already created in the image.
By mounting over it, you will mask the `internalUser.groovy` we already have there...

Reply:
see all the comments in the related issue: https://github.com/jfrog/charts/issues/287

and the notes initial pr comment:

>  unfortunately the permissions of /tmp/plugins do not allow you to write to /tmp/plugins as non root which is why we are working around that with an emptyDir volume mount. _making the behaviour such that the only plugins that are loaded via secrets are loaded_.

we need /tmp/plugins created from the artifactory image to be writable to appropriately copy into the folder.  If you are using these plugins via secrects , just know/assume that those are the only plugins noted.   I think this behavior is better than the current one of being broken

Reply:
Right! My mistake. 
So to cover this, we need to specify this very clearly in the `README.md` and also make a note about `internalUser.groovy`, which is missing that means the auto nginx config update will not work due to its absence.

Anyway - I added the permissions fix to Artifactory Dockerfile, but it will take a few minor releases for it to be public.

Reply:
updated